sites/tv.trueid.net/tv.trueid.net.config.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
   site: 'tv.trueid.net',
   days: 1,
   url({ channel }) {
-    return `https://tv.trueid.net/_next/data/1380644e0f1fb6b14c82894a0c682d147e015c9d/th-${channel.lang}.json?channelSlug=${channel.site_id}&path=${channel.site_id}`
+    return `https://tv.trueid.net/_next/data/9d13441bf2f87fe680d62c50845f1037632855a3/th-${channel.lang}.json?channelSlug=${channel.site_id}&path=${channel.site_id}`

```

Comment:
Update also url in test, otherwise it always fail.

Reply:
Fixed the test as well

Reply:
Instead of manually update these build id, you can fetch it automatically using this examples:

https://github.com/iptv-org/epg/blob/eff2f6ba15330e0a951b105dedda78b1a4b12fdb/sites/tvplus.com.tr/tvplus.com.tr.config.js#L21-L36
https://github.com/iptv-org/epg/blob/eff2f6ba15330e0a951b105dedda78b1a4b12fdb/sites/tvplus.com.tr/tvplus.com.tr.config.js#L86-L99